src/main/cpp/Robot.cpp

```diff
@@ -78,6 +78,7 @@ void Robot::TeleopInit() {
   }
 
   m_container.StartIdling();
+  m_container.StopMotors();
   // m_container.ResetPose();
```

Comment:
Still want this?

Reply:
Yes, if the auto doesn’t finish in 15s it needs to stop in teleop init

Reply:
Was referring to the commented-out call to `ResetPose`